changelog/262.trivial.rst

```diff
@@ -0,0 +1,3 @@
+Fix docstring of ``NDCubeSequencePlotMixin.plot_as_cube`` to to get rid of
```

Comment:
```suggestion
Fix  docstring formatting to help docs build.
```

Reply:
This is quite verbose for a changelog.  Suggest condensing it to the one-liner, above.